### PR TITLE
Initial version

### PR DESCRIPTION
**Pull request by sethyates**

### CODE REVIEW

**Comment by sethyates** [comment]
@Ambroos - mind having a look at this as well?  This is the `htmlParser` split out from `postscribe`, with your ES6 conversion.

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+# Editor artifacts
+.idea/
+.*.swp
+.*.un~
+*.iml
+
+# NPM artifacts
+node_modules/
+npm-*.log
```
Comment (jnewman) [inline .gitignore]: Should add the npm cache as well, right?

`npm-*`

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+# Generated files
+.idea/
```
Comment (jnewman) [inline .gitignore]: Duplicate

File: README.md [hunk 3]
```diff
@@ -1 +1,157 @@
...
+To test the code locally, you can use:
+
+```console
+npm test
+```
+
+Or, open the [test.html](./test/test.html) file in your web browser:
+
+```console
+open test/test.html
```
Comment (jnewman) [inline README.md]: This file doesn't exist

File: bower.json [hunk 4]
```diff
@@ -0,0 +1,17 @@
+{
+  "name": "prescribe",
+  "description": "Tiny, forgiving HTML parser",
+  "license": "MIT",
+  "homepage": "https://krux.github.io/prescribe/",
```
Comment (jnewman) [inline bower.json]: Should we put this up?

File: bower.json [hunk 5]
```diff
@@ -0,0 +1,17 @@
...
+  "keywords": [
+    "html",
+    "parser"
+  ],
+  "main": "dist/prescribe.js",
```
Comment (jnewman) [inline bower.json]: Can we use src/main.js for this?

File: gulpfile.babel.js [hunk 6]
```diff
@@ -0,0 +1,153 @@
...
+  ` * @copyright ${new Date().getFullYear()} Krux Digital, Inc`,
+  ' */',
+  ''
+].join('\n');
+
+gulp.task('clean', () => {
+  return del.sync([`${DEST}/**`]);
+});
+
+const LINTABLE_PATTERN = ['*.js', 'src/**/*.js', 'test/unit/*.spec.js'];
```
Comment (jnewman) [inline gulpfile.babel.js]: Should we just lint `**/*.js`?

File: gulpfile.babel.js [hunk 7]
```diff
@@ -0,0 +1,153 @@
...
+}
+
+gulp.task('test', karma('karma'));
+
+gulp.task('test:cross-browser', karma('karma-sauce'));
+
+gulp.task('test:tdd', karma('karma', false));
+
+gulp.task('tdd', ['test:tdd'], () => {
+  gulp.watch(['*.js', 'src/**', 'test/**'], ['test:tdd']);
```
Comment (jnewman) [inline gulpfile.babel.js]: I'd skip `*.js` here; it confusing to see the tests reload, but the config changes not take effect. (Assuming gulp doesn't reload the gulpfile in these watch statements.)

File: package.json [hunk 8]
```diff
@@ -0,0 +1,127 @@
...
+      "web": "http://creationeer.me"
+    },
+    {
+      "name": "Joshua Newman",
+      "web": "https://github.com/jnewman"
+    },
+    {
+      "name": "Seth Yates",
+      "web": "https://github.com/sethyates"
+    }
```
Comment (jnewman) [inline package.json]: Should add @Ambroos here since he added to this pull.

File: saucelabs.config.babel.js [hunk 9]
```diff
@@ -0,0 +1,185 @@
...
+    browserName: 'chrome',
+    version: '48.0'
+  },
+  el_capitan_chrome_38: {
+    base: 'SauceLabs',
+    platform: 'OS X 10.11',
+    browserName: 'chrome',
+    version: '38.0'
+  },
+  el_capitan_firefox_44: {
```
Comment (jnewman) [inline saucelabs.config.babel.js]: I'd ditch the old firefoxes.

File: test/helpers/testParse.js [hunk 10]
```diff
@@ -0,0 +1,35 @@
+import HtmlParser from '../../src/HtmlParser';
+
+/**
+ * Test parsing
+ *
+ * @param {HtmlParser|string} s A parser to continue or a string tos tart with
```
Comment (jnewman) [inline test/helpers/testParse.js]: typo `to start`

**Comment by jnewman** [comment]
The tests aren't going to work on IE8 b/c chai doesn't support it. I'm thinking we should move to expect.js

**Comment by sethyates** [comment]
Bah.  I knew that...  Force of habit using chai...

File: gulpfile.babel.js (hunk 6 above)
Comment (sethyates) [inline gulpfile.babel.js]: No as that picks up `node_modules`.

File: bower.json (hunk 5 above)
Comment (sethyates) [inline bower.json]: Not sure.  I'm not really a `bower` user, so input welcomed.

File: bower.json (hunk 4 above)
Comment (sethyates) [inline bower.json]: No, moved to the GitHub repo address.

File: README.md (hunk 3 above)
Comment (sethyates) [inline README.md]: Removed.

File: .gitignore (hunk 2 above)
Comment (sethyates) [inline .gitignore]: Removed.

File: .gitignore (hunk 1 above)
Comment (sethyates) [inline .gitignore]: Added.

File: gulpfile.babel.js (hunk 7 above)
Comment (sethyates) [inline gulpfile.babel.js]: Done.

File: package.json (hunk 8 above)
Comment (sethyates) [inline package.json]: Added.

File: saucelabs.config.babel.js (hunk 9 above)
Comment (sethyates) [inline saucelabs.config.babel.js]: Ditched.

File: test/helpers/testParse.js (hunk 10 above)
Comment (sethyates) [inline test/helpers/testParse.js]: Fixed.

File: README.md [hunk 11]
```diff
@@ -1 +1,157 @@
...
+```console
+npm install
+```
+
+### Building
+
+To build the code, run
+
+```console
+npm build
```
Comment (jnewman) [inline README.md]: Should this be `npm run build`?

File: gulpfile.babel.js (hunk 6 above)
Comment (jnewman) [inline gulpfile.babel.js]: Ah, right.

**Comment by jnewman** [comment]
:+1:

**Pull request merged**